package.json

```diff
@@ -31,14 +31,14 @@
     "version-nightly": "bash scripts/should-nightly-run.sh && npm version prerelease --no-git-tag-version --preid=$(git rev-parse HEAD)",
     "postversion": "bash scripts/push-if-git.sh",
     "bundlesize": "yarn build && yarn build:dead-code-elimination && size-limit",
-    "postinstall": "npx yarn-deduplicate"
+    "postinstall": "npx yarn-deduplicate && yarn --ignore-scripts"
```

Comment:
Are we ignoring postinstall script for dependencies? Any specific reason? 🤔

Reply:
only for postinstall - this is to stabalize the yarn lock after de-duping (and so it doesnt result in an infinite loop)

unfortunately there can be instances where running yarn after yarn de-dupe will result in the yarn lock file updating again